README.md

```diff
@@ -1076,6 +1076,46 @@ _You can enable the following settings in Xcode by running [this script](resourc
 
   </details>
 
+* <a id='omit-single-expression-return'></a>(<a href='#omit-single-expression-return'>link</a>) **Omit the `return` keyword from single-expression functions, accessors, and closures.** [![SwiftFormat: redundantReturn](https://img.shields.io/badge/SwiftFormat-redundantReturn-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#redundantReturn)
```

Comment:
Maybe:

```suggestion
* <a id='omit-single-expression-return'></a>(<a href='#omit-single-expression-return'>link</a>) **Omit the `return` keyword from single expressions when not required by the language.** [![SwiftFormat: redundantReturn](https://img.shields.io/badge/SwiftFormat-redundantReturn-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#redundantReturn)
```

I don't think we need to be super specific about function, accessors and closures here

Reply:
@fdiaz This is a pedantic distinction, but in the [Swift grammar](https://docs.swift.org/swift-book/ReferenceManual/zzSummaryOfTheGrammar.html) the [`return`](https://docs.swift.org/swift-book/ReferenceManual/Statements.html#ID444) keyword isn't part of the [`expression`](https://docs.swift.org/swift-book/ReferenceManual/Expressions.html).

Here's the relevant grammar for a `function-body` (simplified to only show the relevant single-expression cases):
```
function-body → code-block

code-block → { statement }

statement → expression
statement → control-transfer-statement

control-transfer-statement → return-statement

return-statement → return expression 
```

In this grammar, `function-body` can either be `{ expression }` or `{ return expression }`. 

So within the Swift grammar, it's not strictly accurate to say that we can omit the `return` keyword _from an expression_. This grammar allows us to omit the `return` keyword from a single-expression function.

Reply:
Could we simplify to this?

> **Omit the `return` keyword when not required by the language.**

Reply:
@bachand That sounds reasonable. There's existing precedent in the style guide for this sort of "required by the language" wording:

> Don't use self unless it's necessary for disambiguation or required by the language. ([link](https://github.com/airbnb/swift#omit-self))

Reply:
I've been working on this style guide a long time @calda  😉